packages/fuels-core/src/code_gen/custom_types_gen.rs

```diff
@@ -35,6 +38,8 @@ pub fn expand_internal_struct(prop: &Property) -> Result<TokenStream, Error> {
     // `MyStruct::new_from_tokens()`.
     let mut args = Vec::new();
 
+    let mut contains_enums = false;
```

Comment:
How about we eliminate the need for this `contains_enums`? 

I don't think we need the `Default` derive at all (personally couldn't find any use case). So this https://github.com/FuelLabs/fuels-rs/blob/087425f3d69afe5df790f0d441078b198ade40be/packages/fuels-core/src/code_gen/custom_types_gen.rs#L134-L138 could be simplified and we could remove the `contains_enums` variable.